CHANGELOG.md

```diff
@@ -4,8 +4,82 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 
-## [Unreleased]
+## [0.6.0] - 2021-01-XX
+### Compatibility Changes:
+- support for CUDA 11, 11.1, and 11.2 #1076 #1086 #1147 #1231
+- support for CMake 3.18.0 and 3.19.0 #1087 #1217
+- set minimal HIP version to 3.5 #1110
+- remove CMake HIP module shipped with alpaka #1189
+- set HIP-clang as default compiler for HIP #1113
+- support for NVCC + VS 2019 #1121
+- remove support for CUDA 11.0 with MSVC 2019 #1227
+- support for boost-1.74.0 #1142
+- explicitly require backends and do not enable them by default #1111
+- remove support for Xcode 11.1 #1206 
+- support Xcode 11.21 - 12.2.0 #1206
+- update to Catch 2.13.3 #1215
+
+### Bug Fixes:
+- apply some clang-tidy fixes #1044
+- fix CUDA/HIP accelerator concept usage #1064
+- fix INTEL compiler detection #1070
+- CMake: build type CXX flag not passed to nvcc #1073
+- work around Intel ICE when using std::decay on empty template parameter packs #1074
```

Comment:
What is Intel ICE?

Reply:
ICE is the official compiler speak for Internal Compiler Error.

Reply:
Now that you explained it, I think I have heard that before! Thx!